### PR TITLE
SI-8875 showCode should print all class constructor modifiers.

### PR DESCRIPTION
**Pull request by gourlaysama**
showCode used to print nothing when the only modifier was a change in
visibility scope (i.e. no flags but privateWithin is set).

### CODE REVIEW

**Comment by gourlaysama** [comment]
PLS REBUILD ALL

**Comment by dragos** [comment]
PLS REBUILD/pr-scala@037f295

**Comment by dragos** [comment]
PLS REBUILD/pr-scala@037f295

**Comment by dragos** [comment]
PLS REBUILD/pr-scala@ee5a5d1

**Comment by gourlaysama** [comment]
PLS REBUILD ALL

File: src/reflect/scala/reflect/internal/Printers.scala [hunk 1]
```diff
@@ -761,7 +761,7 @@ trait Printers extends api.Printers { self: SymbolTable =>
             val build.SyntacticClassDef(_, _, _, ctorMods, vparamss, earlyDefs, parents, selfType, body) = cl
 
             // constructor's modifier
-            if (ctorMods.hasFlag(AccessFlags)) {
+            if (ctorMods.hasFlag(AccessFlags) || ctorMods.privateWithin != tpnme.EMPTY) {
```
Comment (retronym) [inline src/reflect/scala/reflect/internal/Printers.scala]: `ctorMods.hasAccessBoundary` is a higher level way of saying the same thing.

**Comment by adriaanm** [comment]
Triggering rebuild now that IDE validation should be fixed. Sorry about all the noise.

File: src/reflect/scala/reflect/internal/Printers.scala (hunk 1 above)
Comment (gourlaysama) [inline src/reflect/scala/reflect/internal/Printers.scala]: Thanks, I missed that one. Fixed.

**Comment by retronym** [comment]
LGTM

**Pull request merged**